### PR TITLE
Add docs for WEB underfs

### PR DESCRIPTION
**Pull request by IT-DONG**
add brief docs for both EN and CN versions

### CODE REVIEW

**Comment by IT-DONG** [comment]
@calvinjia @apc999

File: docs/cn/ufs/WEB.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+./bin/alluxio format
+./bin/alluxio-start.sh local
+```
+
+要验证Alluxio是否正在运行，你可以访问**[http://localhost:19999](http://localhost:19999)**，或者查看`logs`下的日志。
+
+接着，你可以运行一个简单的示例程序：
+
+```bash
+./bin/alluxio runTests
```
Comment (apc999) [inline docs/cn/ufs/WEB.md]: this will not work as web UFS is read only. Removing runTests?

Comment (apc999) [reply docs/cn/ufs/WEB.md]: how about using mounting `https://downloads.alluxio.io/downloads/files/` as an example?

Comment (IT-DONG) [reply docs/cn/ufs/WEB.md]: > this will not work as web UFS is read only. Removing runTests?

Updated!

Comment (IT-DONG) [reply docs/cn/ufs/WEB.md]: > how about using mounting `https://downloads.alluxio.io/downloads/files/` as an example?

Done!

Comment (IT-DONG) [reply docs/cn/ufs/WEB.md]: @apc999  Done!

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please

**Pull request merged**